gprofiler/merge.py

```diff
@@ -35,7 +35,7 @@
 # ffffffff81082227 mmput+0x57 ([kernel.kallsyms])
 # 0 [unknown] ([unknown])
 # 7fe48f00faff __poll+0x4f (/lib/x86_64-linux-gnu/libc-2.31.so)
-FRAME_REGEX = re.compile(r"^\s*[0-9a-f]+ (.*?) \((.*)\)$")
+FRAME_REGEX = re.compile(r"^\s*[0-9a-f]+ (.*?) \(\[?/?(?:[^/]+/)*(.*?)\]?\)$")
```

Comment:
I think it's more readable if you do the parsing in the Python side in not in the regex side

Reply:
Expression was simplified.
Also, there's no parsing needed since we depart from basename approach for now.

Reply:
👍